addons/ai_apps/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_ai_composer,ai.composer,model_ai_composer,base.group_user,1,1,1,1
```

Comment:
I don't think all composers are supposed to be readable (and writable) by everyone else.
Missing record rules?